pkg/scripts/testdata/TestMigrateToContainerd-simple.golden

```diff
@@ -4,30 +4,6 @@ sudo systemctl stop kubelet
 sudo docker ps -q | xargs sudo docker stop || true
 sudo docker ps -qa | xargs sudo docker rm || true
 
-cat <<EOF | sudo tee /etc/containerd/config.toml
-version = 2
-
-[metrics]
-address = "127.0.0.1:1338"
-
-[plugins]
-[plugins."io.containerd.grpc.v1.cri"]
-[plugins."io.containerd.grpc.v1.cri".containerd]
-[plugins."io.containerd.grpc.v1.cri".containerd.runtimes]
-[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
-runtime_type = "io.containerd.runc.v2"
-[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc.options]
-SystemdCgroup = true
-[plugins."io.containerd.grpc.v1.cri".registry]
-[plugins."io.containerd.grpc.v1.cri".registry.mirrors]
-[plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
-endpoint = ["https://registry-1.docker.io"]
-EOF
-
-cat <<EOF | sudo tee /etc/crictl.yaml
-runtime-endpoint: unix:///run/containerd/containerd.sock
-EOF
-
```

Comment:
What happened with this config? Where is it generated now?

Reply:
All because of missing dot, to pass render context down to the template! :exploding_head: